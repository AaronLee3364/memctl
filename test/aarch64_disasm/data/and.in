fffffff0070c0000
12061423 8a0a016c ea827c10 2a140109
aa0ac108 3219006b ea0600df aa1c03e0
321c0fe3 121f0115 0a290294 6a09011f
4a140108 4a0b2108 2a2003e0 2a2a0129
6a29011f 4a280128 ea018f3d b201f3e9
