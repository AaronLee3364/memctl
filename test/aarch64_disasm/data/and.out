fffffff0070c0000  AND     W3, W1, #0xfc000000
fffffff0070c0004  AND     X12, X11, X10
fffffff0070c0008  ANDS    X16, X0, X2, ASR #31
fffffff0070c000c  ORR     W9, W8, W20
fffffff0070c0010  ORR     X8, X8, X10, LSL #48
fffffff0070c0014  ORR     W11, W3, #0x80
fffffff0070c0018  TST     X6, X6
fffffff0070c001c  MOV     X0, X28
fffffff0070c0020  MOV     W3, #0xf0
fffffff0070c0024  AND     W21, W8, #0x2
fffffff0070c0028  BIC     W20, W20, W9
fffffff0070c002c  TST     W8, W9
fffffff0070c0030  EOR     W8, W8, W20
fffffff0070c0034  EOR     W8, W8, W11, LSL #8
fffffff0070c0038  MVN     W0, W0
fffffff0070c003c  ORN     W9, W9, W10
fffffff0070c0040  BICS    WZR, W8, W9
fffffff0070c0044  EON     W8, W9, W8
fffffff0070c0048  ANDS    X29, X25, X1, LSL #35
fffffff0070c004c  MOV     X9, #0xaaaaaaaaaaaaaaaa
